sklearn/decomposition/tests/test_nmf.py

```diff
@@ -237,7 +237,7 @@ def test_non_negative_factorization_consistency(init, solver, regularization):
     A[:, 2 * np.arange(5)] = 0
 
     W_nmf, H, _ = non_negative_factorization(
-        A, init=init, solver=solver,
+        A, init=init, solver=solver, update_H=True,
```

Comment:
Just to clarify, this change was not really needed because it was already the default value for the `update_H` param in master and the goal is just to make the test more explicit, right?

I assume this PR should not change the behavior of the public methods and therefore this test would not need this to pass.

Reply:
Thanks @ogrisel for checking, I have reverted this change: it was related to my version of the code which has been modified since then.